src/main/java/org/springframework/data/redis/connection/lettuce/LettuceReactiveZSetCommands.java

```diff
@@ -189,28 +190,28 @@
 				if (command.isWithScores()) {
 
 					result = cmd
-							.zrangeWithScores(command.getKey(), command.getRange().getLowerBound().getValue().orElse(null),
-									command.getRange().getUpperBound().getValue().orElse(null))
+							.zrangeWithScores(command.getKey(), command.getRange().getLowerBound().getValue().orElse(0L),
```

Comment:
I do think that the -inf at https://redis.io/commands/zrangebyscore should be translated to Long.MIN_VALUE here.